package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "addonslist",
+  "version": "1.0.0",
+  "description": "A Vue.js project",
+  "author": "seifeddineDaoud <sdaoud@exoplatform.com>",
+  "private": true,
+  "scripts": {
+    "dev": "webpack-dev-server --inline --progress --config build/webpack.dev.conf.js",
+    "start": "npm run dev",
+    "build": "node build/build.js"
```

Comment:
Do we really need this build/build.js file for building the app ? It looks like a complex build script to build such a basic app.
Can't we just use webpack directly, as we did in chat application (https://github.com/exo-addons/chat-application/blob/develop/application/package.json#L15) ?

Reply:
Seif used the recommended way to use vueJs with the vue-cli, so it's the easy way for him to start working with vueJS. IMO, using webpack directly it's more complex

Reply:
You mean that all these build files (build.js, check-versions.js, utils.js, vue-loader.conf.js, dev.env.js, ...) are generated by vue-cli (I did not try, I have to give it a try) ?

Reply:
Yes

Reply:
:) I agree